packages/components/src/components/hds/side-nav/portal/target.ts

```diff
@@ -41,7 +41,11 @@ export default class HdsSideNavPortalTarget extends Component<HdsSideNavPortalTa
   @tracked lastPanelEl: Element | undefined;
 
   static get prefersReducedMotionOverride(): boolean {
-    return Ember.testing;
+    if (macroCondition(isTesting())) {
```

Comment:
I tried just doing `return macroCondition(isTesting())` and the build failed 🤷🏽

Reply:
what about this? does it fail as well?
```
const isTesting = macroCondition(isTesting());
return isTesting;
```

(also, what does `macroCondition(isTesting())` return?)